ts/components/wallet/CreditCardComponent.tsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * This component is used to display a credit card
+ * with its information.
+ */
+// TODO: this component will be substantially modified, in order to:
+// - have a grid-only layout that does not rely on margins and other ungodly things
+// - be used throughout the app whenever needed (as of now, CreditCardComponent can
+//   only be used when displaying the list of credit cards, with the rest being
+//   cropped pictures)
+// @https://www.pivotaltracker.com/story/show/157422715
+
+import * as React from "react";
+
+import { Body, Card, Icon, Text } from "native-base";
+import { Image, StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+
+import I18n from "../../i18n";
+
+import {
+  CreditCard,
+  CreditCardType,
+  getCardType
+} from "../../types/CreditCard";
+
+import color from "color";
+import variables from "../../theme/variables";
+
+type Props = Readonly<{
+  item: CreditCard;
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+export const CreditCardStyle = StyleSheet.create({
+  largeTextStyle: {
+    fontSize: variables.fontSize4,
+    fontFamily: "Roboto Mono"
```

Comment:
I have checked on both Android and iOS. The font is not working on Android

![android](https://user-images.githubusercontent.com/30595520/40366688-1173412e-5dd8-11e8-985f-63ba18b774f7.jpg)

![simulator screen shot - iphone 6 - 2018-05-22 at 15 16 05](https://user-images.githubusercontent.com/30595520/40366696-1670e104-5dd8-11e8-9925-5edc3709e023.png)

Check the `ts/theme/fonts.ts` file.

Reply:
True that! Would you recommend updating `ts/theme/fonts.ts`'s `makeFontStyleObject` to support multiple fonts or should I just use `Platform.select`, being this the only place where Roboto Mono is used?

Reply:
(I'm pushing the solution implementing the first option now, you can have a look at that and let me know)